packages/zowe-explorer/src/webviews/package.json

```
@@ -11,7 +11,7 @@
     "preview": "vite preview",
     "fresh-clone": "yarn clean && rimraf node_modules",
     "clean": "gulp clean",
-    "package": "echo \"webviews: nothing to package.\"",
+    "package": "node -e \"process.exit(+fs.existsSync(path.join(__dirname, 'dist')))\" && pnpm build || echo \"webviews: already packaged.\"",
```

Clever way to turn the boolean output of `fs.existsSync` into an exit code 😋  Curious if this would also work:
```suggestion
    "package": "node -e \"fs.accessSync(path.join(__dirname, 'dist'))\" && pnpm build || echo \"webviews: already packaged.\"",
```

---

Thanks for the suggestion! I think `process.exit` is required in order to propagate the return value of the function call. With the suggested change, the script always runs the `pnpm build` command. Also, it seems that [`accessSync` returns `undefined` upon success](https://nodejs.org/api/fs.html#fsaccesssyncpath-mode) and throws an error if the checks fail - so I'll have to adjust the script a bit more to work with that function.